testcases/Test_build.py

```diff
@@ -0,0 +1,143 @@
+#test_build.py: to make build of the repo and test it through exit code
+
+#!/usr/bin/env python3
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2023
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import json
+# import requests
+import os
+import uuid
+from datetime import datetime
+import re
+import unittest
+import os
+from urllib.parse import urlparse
+from enum import Enum
+import subprocess
+import OpTestConfiguration
+import OpTestLogger
+import configparser
+import sys
+from testcases import Build_bisector
```

Comment:
can you please help me with API document as not able locate , providing documentation URL is real help

Reply:
sorry for the noise , got it